packages/material/scss/pager/_layout.scss

```diff
@@ -0,0 +1,47 @@
+@import "~@progress/kendo-theme-default/scss/pager/layout";
+
+@include exports ('pager/layout/material') {
+    $pager-nav-padding: 6px !default;
+    $pager-elements-margin: 8px !default;
+    $pager-elements-margin-lg: 32px !default;
+    $pager-page-link-size: 36px !default;
+
+    .k-pager-wrap {
+        padding: $pager-padding-y $pager-padding-x;
+
+        .k-icon {
+            width: $icon-size;
+            height: $icon-size;
+            line-height: $line-height-lg;
```

Comment:
What is the reasoning behind this rule? With it, the icons are misaligned vertically:

![image](https://user-images.githubusercontent.com/90405/36108887-7f88003c-1026-11e8-90ed-aa275ec8f65a.png)

Without this rule, this issue is resolved.